packages/toggleInput/components/ToggleInput.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
```

Comment:
This component would be excessive if we were just using it for the Checkbox, but this will make the radio input and toggle switch input components go much faster.